### PR TITLE
Feature/fix orpml processing

### PR DESCRIPTION
**Pull request by jllyxtch**
# Description, Motivation, and Context #
Changing the ORPML functions to more closely align to the schema

# How Has This Been Tested? #
Multiple documents for each format have been ingested into the pipeline

# Types of change/s #
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that does not add functionality)
- [ ] Documentation Update

# Checklist: #
- [x] I merged current develop to this branch.
- [ ] My change requires a change to the documentation (confluence test paths, code style etc.)
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to tests.
- [ ] I have updated tests accordingly.
- [ ] I unit tested new methods.
- [ ] I checked linter on local.
- [ ] I checked unit tests on local.

# Is additional manual testing needed? #
...

# Files with major changes / what is worth paying more attention to? #
...

### CODE REVIEW

File: data/orpml/example_input_2.xml
Comment (imanehaf) [inline data/orpml/example_input_2.xml:1]: Is it necessary to push up example files to the repo?

File: data/orpml/orpml.xsd
Comment (imanehaf) [inline data/orpml/orpml.xsd:1]: the schema file should probably live somewhere else (S3?) if necessary for validation

**Review by imanehaf — APPROVED** [review]
Other than the data comments, LGTM!

File: data/orpml/example_input_2.xml
Comment (jllyxtch) [reply data/orpml/example_input_2.xml:1]: No it's not, this is temporary

File: data/orpml/orpml.xsd
Comment (jllyxtch) [reply data/orpml/orpml.xsd:1]: Agreed, this is temporary

**Pull request merged**